build.sbt

```
@@ -63,13 +63,13 @@ lazy val onnx = project
 
 lazy val runtime = project
   .enablePlugins(ScalaNativePlugin, ScalaNativeBrewedConfigPlugin)
-  .dependsOn(ir)
+  .dependsOn(ir, onnx)
```

Is this just for testing? I think the runtime shouldn't directly depend on ONNX.

```suggestion
  .dependsOn(ir, onnx % Test)
```